web_src/fomantic/build/semantic.css

```diff
@@ -31802,7 +31802,7 @@ Floated Menu / Item
   opacity: 1;
 }
 
-/* Icon Glyph */
+/* Icon Gylph */
```

Comment:
Why such change?

Reply:
These changes come from running make fomantic. 

The spelling changes should never have been made in the first place as they're essentially vendored files.